OMIEData/FileReaders/marginal_price_file_reader.py

```diff
@@ -124,14 +125,20 @@ def _process_line(self, date: dt.date, concept: DataTypeInMarginalPriceFile, val
         locale.setlocale(locale.LC_NUMERIC, MarginalPriceFileReader.__localeInFile__)
 
         for i, v in enumerate(values, start=1):
-            if i > 24:
+
+            if i > 25:
                 break # Jump if 25-hour day or spaces ..
             try:
                 f = multiplier * locale.atof(v)
             except:
+
                 if i == 24:
                     # Day with 23-hours.
-                    result[key_list[25]] = result[key_list[24]]
+                    result[key_list[25]] = np.nan
+                    result[key_list[26]] = np.nan
+                elif i == 25:
+                    # Day with 25-hours.
```

Comment:
I think the comment here should read: # Day with 24-hours.

Reply:
Thank you very much for extending the library and uisng it!

Regarding the issues with 23-hour and 25-hours days: when I first wrote the code, I though on providing the simplest solution to simply the post-processing of the data. Now after this work I think your solution is more precise.
Again, thanks!